modules/test/playwright/tests/client-extension-web/editorConfigContributor.spec.ts

```diff
@@ -6,39 +6,34 @@
 import {expect, mergeTests} from '@playwright/test';
 
 import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {isolatedSiteTest} from '../../fixtures/isolatedSiteTest';
 import {loginTest} from '../../fixtures/loginTest';
+import {liferayConfig} from '../../liferay.config';
+import getRandomString from '../../utils/getRandomString';
+import getPageDefinition from '../layout-content-page-editor-web/utils/getPageDefinition';
+import getWidgetDefinition from '../layout-content-page-editor-web/utils/getWidgetDefinition';
 import {clientExtensionsPageTest} from './fixtures/clientExtensionsPageTest';
+import {editorSamplePageTest} from './fixtures/editorSamplePageTest';
 import {newEditorConfigContributorPageTest} from './fixtures/newEditorConfigContributorPageTest';
 
 export const test = mergeTests(
 	apiHelpersTest,
 	clientExtensionsPageTest,
+	editorSamplePageTest,
+	featureFlagsTest({
+		'LPS-178052': true,
+		'LPS-186870': true,
```

Comment:
`LPS-178052` is [required](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/helpers/HeadlessDeliveryApiHelper.ts#L18) by headless API.